### ISSUE TITLE
Document language missing

### ISSUE THREAD

**Issue by shiffman** [CodingTrain/website#792]
<img width="270" alt="shows document error message" src="https://user-images.githubusercontent.com/191758/47237271-8ab8c280-d3ac-11e8-9fd8-9b163f025e4b.png">

**Comment by nilsweb** [comment]
That should be an easy fix. I'll add it shortly!
